### DISCUSSION TITLE
Cannot clone the source

### DISCUSSION THREAD

**Discussion by sum2012** [project-jedi/jcl#131] [Q&A] [answered]
I am using windows 7.Neither Git Desktop software or git clone https://github.com/project-jedi/jcl.git jcl

Cannot clone this source .Clone JVCL repository is okay

**Comment by obones** [comment]
Without any details on the error you get, it's impossible to help you

**Comment by ronaldhoek** [comment]
Could it be a path depth issue? But, other than that there should be no reason why you can't clone the JCL repo...
But as Win7 is no longer supported... I'd suggest upgrading your OS ;)

**Comment by sum2012** [comment]
Suddenly I now get permissions to clone the source.
Sorry to brother.

**Reply by sum2012** [reply, marked as answer]
I get a email from github
[GitHub] A first-party GitHub OAuth application has been added to your account
A first-party GitHub OAuth application (Git Credential Manager) with gist, repo, and workflow scopes was recently authorized to access your account.
Visit https://github.com/settings/connections/applications/0120e057bd645470c1ed for more information.

To see this and other security events for your account, visit https://github.com/settings/security-log

If you run into problems, please contact support by visiting https://github.com/contact

Thanks,
The GitHub Team
